filedrop/src/filedrop_macos.mm

```diff
@@ -0,0 +1,64 @@
+#if defined(DM_PLATFORM_OSX)
+#include "filedrop.h"
+#include <dmsdk/sdk.h>
+
+#include <AppKit/AppKit.h>
+
+@interface FileDropDelegate: NSView<NSDraggingDestination>
+@end
+
+FileDropDelegate *dragView;
+OnEventCallback draggingCallback;
+
+void FileDrop_PlatformSetEventListener(OnEventCallback callback) {
+    draggingCallback = callback;
+    if (dragView == 0)
```

Comment:
Unf. we can't set delegate for the main window, so it's easy to create a new invisible view with the same size window has and stretch it together with window.